Tests/FunctionalTests/fss-home.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { Browser, BrowserContext, chromium, Page } from 'playwright';
```

Comment:
Do we need  a test for the Terms of Use link ?

Reply:
Hi Greg, There is no "Terms of Use link" on FSS UI therefore no test added for the same.